AprilTagsVision/Robot.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+#
+
+
+import wpilib
+import wpilib.cameraserver
+
+
+#
+# This is a demo program showing the detection of AprilTags. The image is acquired from the USB
```

Comment:
This needs to be a docstring for `MyRobot`

Reply:
Done:
https://github.com/robotpy/examples/pull/100#issuecomment-1873734226